qa-deploy

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env bash
+
+# Bash strict mode
+set -euo pipefail
+
+USAGE="Usage
+===
+
+  $ ./qa-deploy PRODUCTION_URL [STAGING_URL]
+
+Description
+---
+Deploy locally to minikube.
+"
+
+function invalid() {
+    message=${1}
+    echo "Error: ${message}"
+    echo ""
+    echo "$USAGE"
+    exit 1
+}
+
+function run_minikube() {
+    # Run minikube
+    if ! [ minikube ip 2> /dev/null > /dev/null ]; then
```

Comment:
I think this can be written as:
``` bash
if ! minikube ip 2> /dev/null; then
```

Reply:
We want all output to go to `/dev/null` so we need `&>`

```bash
if ! minikube ip &> /dev/null; then
```